tests/Microsoft.Identity.Test.Performance/Helpers/InMemoryCache.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Concurrent;
+using Microsoft.Identity.Client;
+
+namespace Microsoft.Identity.Test.Performance.Helpers
+{
+    public class InMemoryCache
```

Comment:
It would be better to use the MemoryCache from Id.Web here, maybe with an eviction policy.